FrontEnd/src/components/RestorePassword/components/restorepassword/RestorePasswordForm.js

```diff
@@ -0,0 +1,33 @@
+import styles from './RestorePasswordForm.module.css';
+import { RestorePasswordFormContentComponent } from './restorepassword-form/RestorePasswordFormContent';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
```

Comment:
nit: libs import usually is above than custom component/styles. It would be nice to add eslint rule for it

Reply:
Fixed